src/Sylius/Bundle/ProductBundle/Resources/translations/messages.ca.yml

```diff
@@ -10,3 +10,13 @@ sylius:
       meta_description: Metadescripció
     archetype:
       name: Nom
+  form:
```

Comment:
Translations for other languages than English should not be included in pull requests. We use [crowdin.com](https://crowdin.com/project/sylius) for community translations now.